src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
```

Comment:
Using a base font-size of 12px might be too small for some users, especially on larger screens. Consider using a relative unit like `rem` or `em` for better scalability and accessibility.